app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+</section>
+  <section class="col col-space-s col-content-max">
+<%= render TextBlockWithTwoImagesComponent.new(
+heading: "Help your pupils learn new skills",
+colour: "pink",
+image_1_path: "static/images/content/hero-images/0035.jpg",
+image_1_alt: "",
+image_2_path: "static/images/content/hero-images/0037.jpg",
+image_2_alt: "",
+) do %>
+<p>Computing focuses on 3 areas:</p>
+<ul>
+<li>information and communication technology</li>
+<li>digital literacy</li>
+<li>computer science</li>
+</ul>
+<p>You'll inspire pupils to access new fields and teach them new, transferable skills that will open lots of sectors, from engineering to healthcare. Leading them to future careers like software engineering and web design.</p>
+<% end %>
+</section>
+<section class="col col-720">
+<h2>What you'll be teaching</h2>
+<p>At GCSE and A level you'll teach computer science. You will teach themes from the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum</a>, with opportunities to develop your pupils' computational thinking.  </p>
+<p>When teaching 11 to 14 year olds (key stage 3), you could use interactive exercises during your lessons, like creating a robot car. Themes you might also cover are:  </p>
+<ul class="vertical-tags">
+<li class="tag">Algorithms</li>
+<li class="tag">Programming and debugging</li>
+<li class="tag">Creative projects</li>
+<li class="tag">Use of logical reasoning</li>
+<li class="tag">Computer systems</li>
+<li class="tag">Uses of technology</li>
+<li class="tag">Online safety </li>
+</ul>
+<p>When teaching 14 to 16 year olds (key stage 4) computer science as a GCSE qualification. You'll teach pupils about new ways to protect their online privacy and identity. Themes you might also cover are:</p>
+<ul class="vertical-tags">
+<li class="tag">Computational thinking</li>
+<li class="tag">Digital media and information technology</li>
+<li class="tag">Computer networks</li>
+<li class="tag">Cyber security</li>
+<li class="tag">Data representation</li>
+<li class="tag">Digital technology</li>
+</ul>
+<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as a A level qualification and encourage pupils to develop skills to develop websites and prepare them for further learning or employment. Themes you might cover are:</p>
+<ul class="vertical-tags">
+<li class="tag">Abstraction</li>
+<li class="tag">Writing programmes to solve problems</li>
+<li class="tag">Efficiency of algorithms</li>
+<li class="tag">Data structures and the use of databases</li>
+<li class="tag">Systems architecture</li>
+<li class="tag">Computing related mathematics</li>
+</ul>
+
+<p>You can <a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks4-l/units">explore what teaching a computer lesson would be like</a>.</p>
```

Comment:
@caitlin-reidy do we need You can? Could it just start with Explore?

Reply:
@caitlin-reidy and does this also need to say opens in new window? I was looking in the design system and can't find it but I'm sure I've seen something on this for accessibility?